services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -444,6 +444,26 @@ module.exports = {
       return isArray(sectionQuery) ? sectionQuery : [];
     },
 
+    /**
+     * Return the oldest start date for the current section
+     */
+    sectionStartDate: async (content, { input }, { basedb }) => {
+      const { sectionQuery } = content;
+      const { sectionId } = input;
+      const descendantIds = await getDescendantIds(sectionId, basedb);
```

Comment:
This should only happen if section bubbling is enabled. Add that to the input and only get the descendant ids in that case. Check the `websiteScheduledContent` query, match the input key and default value here as well (I believe on by default, but double check please!)

The rest of your logic should stay the same...something like `ids = useSectionBubbling ? await getDescendantIds() : [sectionId];`